### PR TITLE
Team&members - show all users when no team is selected

### PR DESCRIPTION
**Pull request by VasoOppelt**
https://github.com/Einstore/Einstore/issues/53

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -305,6 +305,7 @@ body {
 							{auth && <Layout path="add-api-key/:teamId" body={AddApiKeyRoute} header={Header} />}
 							{auth && <Layout path="team/new" body={AddTeamRoute} header={Header} />}
 							{auth && <Layout path="team/:teamId" body={TeamRoute} header={Header} />}
+							{auth && <Layout path="team/" body={TeamRoute} header={Header} />}
```
Comment (ViliamKopecky) [inline src/App.tsx]: I would remove the trailing `/`

File: src/parts/team.js [hunk 2]
```diff
@@ -41,17 +41,33 @@ export default class Team extends Component {
...
+						})
+					}
+				})
+				.catch((error) => {
+					console.log(error)
+				})
+		} else {
+			window.Einstore.users('')
+				.then((result) => {
+					console.log('Users', result)
```
Comment (ViliamKopecky) [inline src/parts/team.js]: Unnecessary `console.log`

File: src/App.tsx (hunk 1 above)
Comment (VasoOppelt) [reply src/App.tsx]: done

File: src/parts/team.js (hunk 2 above)
Comment (VasoOppelt) [reply src/parts/team.js]: done

**Comment by VasoOppelt** [comment]
@ViliamKopecky done

**Pull request merged**